packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md

```diff
@@ -0,0 +1,322 @@
+4. [Assign users and groups to be pushed to the hub org](#assign-users-and-groups-in-the-org2org-app).
+
+### Add an Org2Org app integration in a spoke org
+
+In the spoke org, where you want to push users and groups to the central hub org, you need to add an instance of the Org2Org app integration by using the [Okta Apps API](/docs/reference/api/apps/#add-basic-authentication-application). This generates an Org2Org app integration instance and the key certificates required to connect to the hub org.
+
+As an Okta admin, make a `POST /api/v1/apps` request to the spoke org with [Okta Org2Org parameters](/docs/reference/api/apps/#add-okta-org2org-application):
+
+| Parameter |  Description/Value   |
+| --------- |  ------------- |
+| `name`  |  `okta_org2org` |
+| `label`  |  Specify a label for this Org2Org app integration |
+| `baseUrl`  |  Specify the base URL of your hub org |
+| `signOnMode`  |  You can set this parameter to any valid value, but if you specify `SAML_2_0`, the Org2Org app signing certificate appears in the Admin Console. |
+
+##### Request example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${spokeApiToken}" \
+-d '{
+  "name": "okta_org2org",
+  "label": "'${spokeOrg2OrgClientLabel}'",
+  "signOnMode": "SAML_2_0",
+  "settings": {
+    "app": {
+        "baseUrl": "https://'${yourHubOktaDomain}'"
+    }
+  }
+}' "https://${yourSpokeOktaDomain}/api/v1/apps"
+```
+
+From the response of your POST request, use the `id` property of the Org2Org app instance to [retrieve the key credentials](/docs/reference/api/apps/#list-key-credentials-for-application) generated for the app with the `GET /api/v1/apps/${id}/credentials/keys` request.
+
+##### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${spokeApiToken}" \
+"https://${yourOktaDomain}/api/v1/apps/${yourOrg2OrgAppId}/credentials/keys"
+```
+
+##### Response example
+
+```json
+[
+    {
+        "kty": "RSA",
+        "created": "2022-01-20T19:50:14.000Z",
+        "lastUpdated": "2022-01-20T19:50:14.000Z",
+        "expiresAt": "2024-01-20T19:50:13.000Z",
+        "kid": "sf-jWwRKMUU55 ... GucHLxIh_-fYLAofB8",
+        "use": "sig",
+        "x5c": [
+            "MIIDqDCCApCgAwIBAgIGAX55CiDiMA0GCSqG ... c5Iuo9j3wpemDSgGapXQ=="
+        ],
+        "x5t#S256": "v-v2V8soFmXuhC ... nrJ4ho-N3P8aASFc",
+        "e": "AQAB",
+        "n": "gIxwqCNkdAb1ioyNBY2boqUCrMj_NSFJAl ... 7dZFiAYF7p_k3XMXOh-hsL_D8FDQ"
+    }
+]
+```
+
+> **Note**: The keys are truncated for brevity.
+
+You need to save the JWKS properties `kty`, `e`, `n`, `use`, and `kid` for use in the corresponding hub org service app.
+
+For each spoke org that you have in your multi-tenant solution, you must create an Org2Org app instance and generate the key credentials that need to be configured in the hub org’s corresponding service app.
```

Comment:
can we update it to `you must create an Org2Org app instance which comes with auto generated key credentials that...`

Reply:
I decided to remove this line. It seems repetitive?